sphinx_automodapi/automodapi.py

```diff
@@ -67,6 +67,11 @@
         Propagates the ``noindex`` flag to autodoc. Use it to avoid duplicate
         objects warnings.
 
+    * ``:sort:``
+        Sort the objects in the module alphabetically. The default is to
+        sort by the order they appear in the module (as specified by `__all__`
```

Comment:
Looking at the actual code below, I see that without `__all__`, our use of `dir(module)` automatically sorts things. So, maybe rewrite as
```
Sort the objects in the module alphabetically also if they are
explicitly given in ``__all__`` (if ``__all__`` is not present, the
objects are found using `dir`, which always gives a sorted list).
```
Actually, writing this suggests that perhaps the default should be to sort also with `__all__` and have a `nosort` option. In that case,
```
Do not sort the objects alphabetically, but use the order given by ``__all__``.
This option has no effect if a module does not have ``__all__``.
```
But probably what you have makes a bit more sense.

Reply:
✅